Dockerfile

```diff
@@ -1,15 +1,21 @@
-FROM ghcr.io/celestiaorg/celestia-node:v0.8.2 AS celestia-node
+FROM ghcr.io/celestiaorg/celestia-app:0.12.2 AS celestia-app
 
-FROM ghcr.io/celestiaorg/celestia-app:0.12.2
+FROM ghcr.io/celestiaorg/celestia-node:v0.8.2
 
-COPY --from=celestia-node /bin/celestia /
+USER root
 
-RUN apk update && apk --no-cache add curl jq libc6-compat
+# hadolint ignore=DL3018
+RUN apk --no-cache add \
+        curl \
+        jq \
+        libc6-compat
 
-COPY entrypoint.sh /
+USER ${USER_NAME}
```

Comment:
Now that you say it, the `USER_NAME` variable is empty, as it's an `ARG` in the celestia-node Dockerfile and not an `ENV`. So we can't use the variable here. I will hardcode `celestia` instead of the variable.
Thank you!